Source/LudoEngine/Piece.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LudoEngine
+{
+    public class Piece
+    {
+        public int ID { get; set; }
+        public bool IsActive { get; set; }
+        public bool HasFinished { get; set; }
+        public int Steps { get; set; }
+        public int Position { get; set; }
+        public string Color { get; set; }
+
+        public Piece(string color)
+        {
+            IsActive = false;
+            HasFinished = false;
+            Steps = 0;
+            Position = 0;
+            Color = color;
+        }
+
+        public override string ToString()
+        {
+            return $"{ID} of {Color} moved {Steps} steps to the position {Position} and has finished:{HasFinished}.";
```

Comment:
Maybe it's enough to just return the int ("Steps") and maybe have this text somewhere in the presentation (console).